x-pack/plugins/maps/public/connected_components/toolbar_overlay/toolbar_overlay.tsx

```diff
@@ -62,6 +73,8 @@ export function ToolbarOverlay(props: Props) {
 
       {toolsButton}
 
+      {featureDrawControl}
```

Comment:
How about moving featureDrawControl after timesliderToogleButon so edit controls get added to the end instead of in the middle